tests/Core/DateTime/FluentDatePickerTests.cs

```diff
@@ -225,4 +225,38 @@ public void FluentDatePicker_ChangeValueWhenDefaultIsNull()
         // Assert
         Assert.Equal(System.DateTime.Parse("2022-03-12"), picker.Instance.Value);
     }
+
+    [Theory]
+    [InlineData(true)]
+    [InlineData(false)]
+    public void FluentDatePicker_DoubleClickToSetTodayInTextField(bool doubleClickToToday)
```

Comment:
Can you add another test to verify the part `OnDoubleClick.HasDelegate`?
The file `_StartCodeCoverage.cmd` can help you to detect which part of code is not tested.

Reply:
A new test `FluentDatePicker_OnDoubleClickEventTriggers` was added for the `OnDoubleClick` event.